src/styles/components/_sticky.scss

```diff
@@ -0,0 +1,15 @@
+.mc-sticky {
```

Comment:
Will also most likely need a z-index so it can layer above content on the page that has z-indexes specified: https://github.com/yankaindustries/mc-components/blob/develop/src/styles/base/_variables.scss#L69